modules/cluster-logging-dashboards-logging.adoc

```diff
@@ -0,0 +1,78 @@
+// Module included in the following assemblies:
+//
+// * logging/cluster-logging-dashboards.adoc
+
+[id="cluster-logging-dashboards-logging_{context}"]
+= About the OpenShift Logging dashboard
+
+The *OpenShift Logging* dashboard contains charts that show details about your Elasticsearch instance at a cluster-level that you can use to diagnose and anticipate problems.
+
+.OpenShift Logging charts
+[options="header"]
+|===
+|Metric|Description
+
+|Elastic Cluster Status
+a|The current Elasticsearch status:
+
+* ONLINE - Indicates that the Elasticsearch instance is online. 
+* OFFLINE - Indicates that the Elasticsearch instance is offline. 
+	
+|Elastic Nodes
+|The total number of Elasticsearch nodes in the Elasticsearch instance.
+
+|Elastic Shards
+|The total number of Elasticsearch shards in the Elasticsearch instance.
+
+|Elastic Documents
+|The total number of Elasticsearch documents in the Elasticsearch instance.
+
+|Total Index Size on Disk
+|The total disk space that is being used for the Elasticsearch indices.
+
+|Elastic Pending Tasks
+|The total number of Elasticsearch changes that have not been executed, such as index creation, index mapping, shard allocation, or shard failure.
+
+|Elastic JVM GC time
+|The amount of time that the JVM spent executing Elasticsearch garbage collection operations in the cluster.
+
+|Elastic JVM GC Rate
+|The total number of times that JVM executed garbage activities per second.
+
+|Elastic Query/Fetch Latency Sum
+a|* Query latency: The average time each Elasticsearch search query takes to execute.
+* Fetch latency: The average time each Elasticsearch search query spends fetching data. 
+
+Fetch latency should take less time than query latency. If fetch latency is consistently increasing, it could indicate slow disks, data enrichment, or large requests with too many results.
```

Comment:
```suggestion
In a healthy cluster, fetch latency takes less time than query latency. If fetch latency is consistently increasing, it might indicate slow disks, data enrichment, or large requests with too many results.
```